pkgs/applications/search/grepcidr/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "grepcidr-${version}";
+  version = "2.0";
+
+  src = fetchurl {
+    url = "http://www.pc-tools.net/files/unix/grepcidr-${version}.tar.gz";
+    sha256 = "1yzpa1nigmmp4hir6377hrkpp0z6jnxgccaw2jbqgydbglvnm231";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp grepcidr $out/bin/
```

Comment:
Please use the default `installPhase`. You'll probably have to set something like
```
installFlags = [ "PREFIX=$out" ];
```

Reply:
checked the makefile. `installFlags = [ "PREFIX=$(out)" ];` should work.